packages/manager/apps/pci-instances/src/pages/instances/datagrid/cell/ActionsCell.spec.tsx

```diff
@@ -20,21 +20,31 @@ describe('Considering the ActionsCell component', () => {
     expect(actionsMenuElement).toBeInTheDocument();
   });
 
-  test('Should render actions menu items with correct labels and hrefs', () => {
+  test('Should render grouped actions menu items with correct labels and hrefs', () => {
     const instance: TInstance = {
       ...mockedInstance,
-      actions: [
-        {
-          name: 'start',
-          enabled: true,
-          link: { path: '/start', isExternal: false },
-        },
-        {
-          name: 'stop',
-          enabled: false,
-          link: { path: '/stop', isExternal: false },
-        },
-      ],
+      actions: new Map([
```

Comment:
I think you should mock from __mocks__ folder or other to avoid changes in the code in the future